src/pages/AppList.svelte

```diff
@@ -1,16 +1,32 @@
 <script>
   const URL = "data/apps.json";
+  let searchTerm = "";
   let apps;
+  let filteredApps;
+
   fetch(URL)
     .then((r) => r.json())
     .then((ret) => {
       apps = ret.sort((a, b) => a.app_id > b.app_id);
+      filteredApps = apps;
     });
+
+  const searchTextChange = () => {
+    const searchApps = (appItem) => appItem.name.includes(searchTerm);
+    filteredApps = apps.filter(searchApps);
+  };
 </script>
 
 <h2>Applications</h2>
+<input
+  class="p-2 mb-4"
+  type="text"
+  bind:value={searchTerm}
+  on:input={searchTextChange}
+  placeholder="Search" />
```

Comment:
The styling we have in the table view component looks a little nicer:

https://github.com/mozilla/glean-dictionary/blob/5e787003843d662079e8c047078c108de8b6d976/src/components/SchemaViewer.svelte#L50

The easiest way to make sure we get this (and that it stays consistent if we ever want to change it) is to refactor the filter box into a seperate component. I'll leave how to do so as an exercise for you to figure out based on the repository layout. 😄 If you need some help, the svelte tutorial is very good: https://svelte.dev/tutorial/basics